docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+* This collection requires some additional dependencies to be installed. To do this, enter the following command.
++
+[source,shell]
+----
+$ pip install rhoas-sdks
+pip install python-dotenv
+----
+NOTE: Use `pip install rhoas-sdks --force reinstall`  to ensure that the most up to date versions of the SDKs are installed.
++
+
+* (Optional) This collection works best when used in a Python virtual environment. To create and activate a Python virtual environment, enter the following command:
++
+[source,shell]
+----
+$ python3 -m venv rhoas
+. rhoas/bin/activate
+----
++
+
+.Procedure
+
+. Open a terminal and enter the following command:
++
+[source,shell]
+----
+$ ansible-galaxy collection install rhoas.rhoas
+----
++
+By default, ansible-galaxy collection install uses https://galaxy.ansible.com as the Galaxy server (as listed in the ansible.cfg file under GALAXY_SERVER). You do not need any further configuration. The install command automatically appends the path `ansible_collections` to the one specified with the `-p` option unless the parent directory is already in a folder called `ansible_collections`.
+. Once installed, you can reference a collection content by its fully qualified collection name (FQCN).
+
+NOTE: The `create_kafka` command in the task needs to be the FQCN of the module for collection development, for example
+`rhoas.rhoas.create_kafka`.
+
+
+[id="proc-using-rhoas-ansible-collection_{context}"]
+=== Using the RHOAS Ansible collection
+
+.Prerequisites
+* You have a Red Hat account.
+* You have an offline token that is accessible to Ansible.
+
+[NOTE]
+The offline token is a refresh token with no expiry and can be used by non-interactive processes to provide an access token for {product-long-rhoas} to authenticate the user. The token is an OpenShift offline token and can be found at https://cloud.redhat.com/openshift/token.
+
+* Two further environment variables are used for the collection to work. These environment variables can be placed in a `.env` file. If neither of these environment variables are set, the collection will default to the URLs shown in the examples.
+- API_BASE_HOST - The base host for the API. This is the base URL for the API. For example, https://api.openshift.com.
+- SSO_BASE_HOST - The base host for the SSO. This is the base URL for the SSO. For example, https://sso.redhat.com/auth/realms/redhat-external.
+
+.Procedure
+
+. Once the collection is installed, documentation for each module can be found by entering `ansible-doc rhoas.rhoas.<module_name>` and replacing <module_name> with the name of the module. The following example outputs the documentation for the `create_kafka` module. It shows the required (denoted by `=`) and optional parameters (denoted by `-`) for the module, along with examples of how to use the module, and the output of the module.
```

Comment:
- [Per IBM style](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o), use "after" or "when" instead of "once". 
- "When the collection is installed, you can find documentation..." (active voice)
- Suggest: "The documentation shows required parameters (denoted by `=`) and optional parameters (denoted by `-`) for the module, as well as examples of how to use the module and the output."